### PR TITLE
README pass for adding detail and polish

### PR DESCRIPTION
**Pull request by tjvantoll**
No code changes. Trying to make the READMEs look a bit nicer. Probably easier to peruse https://github.com/blues/app-accelerators/tree/tj-readme-refresh than to look at the diff.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,54 @@
...
+* 👨‍💻👩‍💻 **Uses open-source firmware.**
+  * We provide the firmware as well as instructions for running it on your devices.
+* 📱 **Provides a way of viewing your device data.**
+  * We show you how to get the data to a useful place to view, which sometimes is a web dashboard, and sometimes is SMS alerts.
+* 📘 **Is free to copy, fork, and build upon.**
+  * This repository uses the Apache 2.0 license, which means each accelerator is free for personal and commercial use.
+
+## The Accelerators
+
+### 1: 🏢 Indoor Floor-Level Tracker
```
Comment (paigen11) [inline README.md:24]: Change the : to a . - the other items in the list have periods behind their numbers

File: better-mouse-trap/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Better Mouse Trap
```
Comment (paigen11) [inline better-mouse-trap/README.md:1]: Is NF3 not getting a full README write up for hardware assembly and firmware configuration here?

**Review by paigen11 — APPROVED** [review]
Minor comments, but otherwise I really like it. Much more polished looking!

File: better-mouse-trap/README.md (hunk 2 above)
Comment (tjvantoll) [reply better-mouse-trap/README.md:1]: Don’t know 🤷‍♂️

For now though I figure this is better than nothing.

File: README.md (hunk 1 above)
Comment (tjvantoll) [reply README.md:24]: 👍

**Pull request merged**